lib/logmsg/tags.c

```diff
@@ -64,12 +64,15 @@ log_tags_get_by_name(const gchar *name)
      In both cases the return value is 0.
    */
   guint id;
+  gpointer key;
 
   g_assert(log_tags_hash != NULL);
 
   g_mutex_lock(&log_tags_lock);
 
-  id = GPOINTER_TO_UINT(g_hash_table_lookup(log_tags_hash, name)) - 1;
+  key = g_hash_table_lookup(log_tags_hash, name);
+  id = GPOINTER_TO_UINT(key) - 1;
+
```

Comment:
Using declarations right at the place of use would make this clearer.

E.g.

```
gpointer key = ...;
guint id = ...;
```

Reply:
Fixed 👍